ios/PurchasesHybridCommon/PurchasesHybridCommonUI/CustomerCenter/CustomerCenterProxy.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  CustomerCenterProxy.swift
+//
+
+#if os(iOS)
+
+import Foundation
+import PurchasesHybridCommon
+import RevenueCat
+import RevenueCatUI
+import UIKit
+
+@available(iOS 15.0, *)
+@available(macOS, unavailable)
+@available(tvOS, unavailable)
+@available(watchOS, unavailable)
+@available(visionOS, unavailable)
+@objc
+public class CustomerCenterProxy: NSObject {
+
+    @objc
+    func createCustomerCenterViewController() -> RCCustomerCenterViewController {
+        return RCCustomerCenterViewController(
+            customerCenterActionHandler: nil
```

Comment:
I guess the action handler will be wrapped in a delegate in future PRs?

Reply:
Yes, we're discussing what's the way to move forward with the action handler so I decided to leave it like this for now. I'll add a comment